src/main/java/com/facebook/openwifirrm/modules/operationelement/HTOperationElement.java

```diff
@@ -0,0 +1,162 @@
+package com.facebook.openwifirrm.modules.operationelement;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+import org.apache.commons.codec.binary.Base64;
+
+import com.facebook.openwifirrm.Utils;
+
+/**
+ * High Throughput (HT) Operation Element, which are potentially present in
+ * wifiscan entries. Details in IEEE 802.11-2020 standard.
+ */
+public class HTOperationElement implements OperationElement {
+
+	protected final byte primaryChannel;
+	protected final byte secondaryChannelOffset;
+	protected final boolean staChannelWidth;
+	protected final boolean rifsMode;
+	protected final byte htProtection;
+	protected final boolean nongreenfieldHtStasPresent;
+	protected final boolean obssNonHtStasPresent;
+	protected final byte channelCenterFrequencySegment2;
+	protected final boolean dualBeacon;
+	protected final boolean dualCtsProtection;
+	protected final boolean stbcBeacon;
+	protected final byte[] basicHtMcsSet;
```

Comment:
why are all of these `protected`? it seems like they should either be `public` or `private`